### ISSUE TITLE
build failure: showVersion is ambiguous

### ISSUE THREAD

**Issue by DanBurton** [kosmikus/lhs2tex#55]
Build failure with GHC 8.2.1 as seen on the Stackage build server:

```
Setup.hs:95:92: error:
    Ambiguous occurrence ‘showVersion’
    It could refer to either ‘Distribution.Simple.showVersion’,
                             imported from ‘Distribution.Simple’ at Setup.hs:6:1-26
                             (and originally defined in ‘Distribution.Version’)
                          or ‘Data.Version.showVersion’,
                             imported from ‘Data.Version’ at Setup.hs:22:1-19
   |
95 |                                                                                Just n  -> (showVersion n,versionBranch n < minPolytableVersion)
   |                                                                                            ^^^^^^^^^^^

```

**Comment by asr** [comment]
I also get the same failure using the same version of GHC.

**Comment by kosmikus** [comment]
Yes, sorry. Thanks for the report. Should be fixed in master and 1.20.
